solc/CommandLineParser.cpp

```diff
@@ -497,6 +504,16 @@ bool CommandLineParser::parseOutputSelection()
 	for (auto&& [optionName, outputComponent]: CompilerOutputs::componentMap())
 		m_options.compiler.outputs.*outputComponent = (m_args.count(optionName) > 0);
 
+	if (m_options.input.mode == InputMode::Assembler && m_options.compiler.outputs == CompilerOutputs{})
+	{
+		// In assembly mode keep the default outputs enabled for backwards-compatibility.
+		// TODO: Remove this (must be done in a breaking release).
```

Comment:
Maybe also create issue?

Reply:
Will do.

Reply:
Actually, I just realized that ewasm is still experimental. So maybe it's not breaking after all?

Reply:
#12230.